src/views/Sandbox/index.tsx

```diff
@@ -1,16 +1,75 @@
 import React, { useEffect } from 'react';
+import { Button } from '@trussworks/react-uswds';
 
 import Header from 'components/Header';
+import SystemProfileHealthCard from 'components/SystemProfileHealthCard';
 
 const Sandbox = () => {
   useEffect(() => {
     document.title = 'Sandbox';
   }, []);
+
   return (
     <div>
       <Header />
       <div className="grid-container">
-        <h1>Sandbox</h1>
+        <div className="desktop:grid-col-10">
+          <h1>Sandbox</h1>
```

Comment:
"Sandbox" (and other text on this page) should probably be put in the `i18n` folder and referenced with the `useTranslation()` hook; not sure if you want to go to that effort for this prototype.

Reply:
Thinking since sandbox is likely to be changed frequently, leaving plaintext in its file isn't too big of a deal (once we demo / use these components, we'll probably just rip them from the sandbox)